docs/sources/panels-visualizations/panel-editor-overview/index.md

```diff
@@ -33,10 +33,32 @@ In the panel editor, you can update all the elements of a visualization includin
 
 ![Panel editor](/media/docs/grafana/panels-visualizations/screenshot-panel-editor-view.png)
 
-To add a panel in a new dashboard click **+ Add visualization** in the middle of the dashboard. To add a panel to an existing dashboard, click **Add** in the dashboard header and select **Visualization** in the dropdown:
+To add a panel in a new dashboard click **+ Add visualization** in the middle of the dashboard. To add a panel to an existing dashboard, click **Add** in the dashboard header and select **Visualization** in the drop-down:
 
 ![Add dropdown](/media/docs/grafana/dashboards/screenshot-add-dropdown-10.0.png)
 
+## Panel menu
```

Comment:
Mainly here to review the Extensions part, but I'd question adding this in a section about the Panel editor overview - since its not part of the editor... I would either have a new page for Panel menu or, if the focus is on enabling users to discover the alerting addition, just to ensure the alerting docs are updated (which they may already be).

Reply:
Thanks for the quick review! 

The alerting docs are already updated so this was a quick and dirty way (given the time crunch) to get something up for the purpose of having a place from which to link to the alerting docs.

While the panel menu isn't specifically part of the panel editor, I'm not sure from a user perspective that there's as much of a distinction as we might feel like there is internally. I definitely don't think this warrants an entire page on its own. I'd like to get an opinion from the Dashboards team as well though. @dprokop?

@Ijin08 - from a UX perspective, do you have thoughts on this?

Reply:
One thing I can do is add some connective tissue. Something like "You can access the panel editor from the panel menu. The panel menu also includes these other options"

Reply:
@sympatheticmoose - @Ijin08 and I talked through placement of this content and we decided that there's a need for a panel overview page that doesn't currently exist. There's not enough time for me to do that before I go on PTO, so we'll go with this treatment for now, but I'll make the change later.

Reply:
sounds good to me 👍